subsys/nrf_security/src/psa_crypto_driver_wrappers.c

```diff
@@ -144,6 +173,14 @@ psa_status_t psa_driver_wrapper_init(void)
 	}
 #endif /* PSA_CRYPTO_DRIVER_TFM_BUILTIN_KEY_LOADER */
 
+#if defined(CONFIG_TRUSTED_STORAGE_BACKEND_AEAD_KEY_DERIVE_FROM_HUK) && \
+	!defined(CONFIG_BUILD_WITH_TFM)
+	status = hw_unique_key_provisioning();
+	if (status != PSA_SUCCESS) {
+		return status;
+	}
+#endif /* CONFIG_TRUSTED_STORAGE_BACKEND_AEAD_KEY_DERIVE_FROM_HUK && !CONFIG_BUILD_WITH_TFM */
+
```

Comment:
This breaks with the pattern of 

psa_driver_wrapper_<feature>() {
#ifdef PSA_CRYPTO_DRIVER_<driver_that_provides_feature>
<driver>_<feature>();
#endif
}

HUK is always implemented by CC3XX right?

We can align with the above mentioned pattern and https://github.com/nrfconnect/sdk-nrf/pull/14184/files#diff-0b8291337394991a5ff58bf61065d475d925374ad0fbb5c01523d5f7643ad29aR152

by introducing a cc3xx_init(); function and calling that here instead of hw_unique_key_provisioning();.

Reply:
That would also allow us to not pollute the common psa_crypto_driver_wrappers.c file with things that are specific to CC3XX.

Reply:
Discussed offline.

We can't put this in a cc3xx_init because HUK is also used by the OBERON_DRIVER.